src/lib/nodes/rate-info-service/RateInfoService.ts

```diff
@@ -5,11 +5,13 @@ import { NODE_LABELS } from '@/lib/nodes/constants'
 import { RateInfoResponse } from '@/lib/nodes/rate-info-service/types/RateInfoResponse'
 import { RateHistoryInfoResponse } from '@/lib/nodes/rate-info-service/types/RateHistoryInfoResponse'
 import { GetHistoryParams } from '@/lib/nodes/rate-info-service/types/GetHistoryParams'
+import type { NodeInfo } from '@/types/wallets'
 
 export class RateInfoService extends Node<AxiosInstance> {
```

Comment:
Don't see the logic for dynamic baseURL: `baseURL: this.preferAltIp ? this.altIp : this.url,`

Reply:
https://github.com/Adamant-im/adamant-im/pull/768/commits/8b50ec259e32c61770fc118c9baf94695c0a8fdd